pkg/controller/controlplane/valuesprovider.go

```diff
@@ -464,12 +469,18 @@ func getConfigChartValues(
 	}
 
 	// Collect config chart values
-	return map[string]interface{}{
+	config := map[string]interface{}{
 		"vpcID":       infraStatus.VPC.ID,
 		"subnetID":    subnet.ID,
 		"clusterName": cp.Namespace,
 		"zone":        subnet.Zone,
-	}, nil
+	}
+
+	if ipFamilies != nil && sets.New[v1beta1.IPFamily](ipFamilies...).Has(v1beta1.IPFamilyIPv6) {
+		config["nodeIPFamilies"] = "ipv6"
+	}
```

Comment:
Does that mean that we cannot have "dual stack" or maybe better that the ipv6 is a superset of the ipv4 ? (given that here we only care about ipv6 and disregard the potential ipv4 family)

Reply:
I changed it, also in [cloud-provider-config.yaml](https://github.com/gardener/gardener-extension-provider-aws/blob/6817edffe6109c78f3b3a4ba194d988040f35498/charts/internal/cloud-provider-config/templates/cloud-provider-config.yaml#L15-L21), so that it should also work in the dual-stack case.